common/app/views/fragments/meta/metaInline.scala.html

```diff
@@ -2,7 +2,16 @@
 
 @import common.{LinkTo, Localisation}
 
-    <div class="content__labels">
+@defining(
+    item.content.tags.tags.exists(_.id == "us-news/us-elections-2016")
+){ isUSElection =>
+    <div class="content__labels @if(isUSElection) {content__labels--us-election}">
```

Comment:
it's bit naff that we have to pepper our generic meta system with this kind of stuff. Although I accept that we just have to do this kind of thing. At the very least, can we wrap it around a switch? Then we can tidy later.